doc/content/guides/connecting-gateways/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Connecting Gateways"
+description: ""
+weight: 3
```

Comment:
We are in a sort of chicken-and-egg situation, but I think the `weight` should be higher.

Reply:
This section should also be mentioned in the getting started in the `Create gateway` section.

Reply:
This should go after **Getting Started** indeed